kernel/public/test-scenes/1.11.uv-tests/game.ts

```diff
@@ -0,0 +1,68 @@
+import { engine, PlaneShape, Entity, Transform, BasicMaterial, Texture, Vector3, BoxShape } from 'decentraland-ecs/src'
+
+const generateDiceUV = (faces : number, offset : number = 0) : number[] => {
+  const sizeDiv = 1.0 / 6.0
+  let uvs : number[] = []
+  for(let i = 0; i < faces; ++i) {
+    uvs = [
+      ...uvs,
+      sizeDiv*(i+1+offset), // lower-right corner
+      0,
+
+      sizeDiv*(i+offset), // lower-left corner
+      0,
+
+      sizeDiv*(i+offset), // upper-left corner
+      1,
+
+      sizeDiv*(i+1+offset), // upper-right corner
+      1,
+    ]
+  }
+  return uvs
+}
+
+//Create material and configure fields
+const material = new BasicMaterial()
+let myTexture = new Texture("dice.png", {
+  samplingMode: 0
+})
+
+material.texture = myTexture
+
+// Create plane shape
+const plane_shape_entity = new Entity()
```

Comment:
For future code, please use consistentCasing mixing_styles_is_not_good. Usually, all C# and TS/JS projects use camelCasing

Reply:
Done, I force-pushed the change, thanks!